quickswap.asm

```diff
@@ -28,6 +28,7 @@ QuickSwap:
 	BRA .store
 
 	.special_swap
+	LDA !INVENTORY_SWAP_2 : ORA #$01 : STA !INVENTORY_SWAP_2
```

Comment:
I'd suggest moving this to after the "BRA .store" call below (line 37 pre-change, line 38 post-change) when we know we're going to process a special item.